src/lwc/geTemplates/geTemplates.js

```diff
@@ -1,44 +1,107 @@
-import { LightningElement, track } from 'lwc';
+import { LightningElement, track, api } from 'lwc';
 import { NavigationMixin } from 'lightning/navigation';
 import getAllFormTemplates from '@salesforce/apex/FORM_ServiceGiftEntry.getAllFormTemplates';
 import deleteFormTemplates from '@salesforce/apex/FORM_ServiceGiftEntry.deleteFormTemplates';
 import cloneFormTemplate from '@salesforce/apex/FORM_ServiceGiftEntry.cloneFormTemplate';
 import getDataImportSettings from '@salesforce/apex/UTIL_CustomSettingsFacade.getDataImportSettings';
 import TemplateBuilderService from 'c/geTemplateBuilderService';
-import { findIndexByProperty } from 'c/utilTemplateBuilder';
+import { showToast, dispatch, handleError } from 'c/utilTemplateBuilder';
 import GeLabelService from 'c/geLabelService';
+
+import FORM_TEMPLATE_INFO from '@salesforce/schema/Form_Template__c';
+import DATA_IMPORT_BATCH_INFO from '@salesforce/schema/DataImportBatch__c';
 import FIELD_MAPPING_METHOD_FIELD_INFO from '@salesforce/schema/Data_Import_Settings__c.Field_Mapping_Method__c';
+import TEMPLATE_LAST_MODIFIED_DATE_INFO from '@salesforce/schema/Form_Template__c.LastModifiedDate';
 
-const actions = [
+const ADVANCED_MAPPING = 'Data Import Field Mapping';
+const DEFAULT_FIELD_MAPPING_SET = 'Migrated_Custom_Field_Mapping_Set';
+const TEMPLATE_BUILDER_TAB_NAME = 'GE_Template_Builder';
+const SUCCESS = 'success';
+const IS_LOADING = 'isLoading';
+const EVENT_TOGGLE_MODAL = 'togglemodal';
+const SAVE = 'save';
+
+const TEMPLATES_LIST_VIEW_NAME = 'Templates';
+const TEMPLATES_LIST_VIEW_SORT_DIRECTION = 'desc';
+const TEMPLATES_LIST_DEFAULT_LIMIT = 10;
+const TEMPLATES_LIST_VIEW_ICON = 'standard:visit_templates';
+const TEMPLATES_TABLE_ACTIONS = [
     { label: 'Edit', name: 'edit' },
     { label: 'Clone', name: 'clone' },
     { label: 'Delete', name: 'delete' }
 ];
 
-const ADVANCED_MAPPING = 'Data Import Field Mapping';
-
-const columns = [
-    { label: 'Template Name', fieldName: 'name' },
-    { label: 'Template Description', fieldName: 'description' },
-    {
-        type: 'action',
-        typeAttributes: { rowActions: actions },
-    },
-];
-
 export default class GeTemplates extends NavigationMixin(LightningElement) {
 
     // Expose custom labels to template
     CUSTOM_LABELS = GeLabelService.CUSTOM_LABELS;
 
     @track templates;
-    @track columns = columns;
-    @track isLoading = true;
-    currentNamespace;
+    @track templatesTableActions = TEMPLATES_TABLE_ACTIONS;
     @track isAccessible = true;
+    @track isLoading = true;
 
     get templateBuilderCustomTabApiName() {
-        return this.currentNamespace ? `${this.currentNamespace}__GE_Template_Builder` : 'GE_Template_Builder';
+        return TemplateBuilderService.alignSchemaNSWithEnvironment(TEMPLATE_BUILDER_TAB_NAME);
```

Comment:
Unfortunately yes for this. The main issue is that we can't get the tab describe info through the schema like we do for objects/fields and hence the TEMPLATE_BUILDER_TAB_NAME constant there.

We then need to align that string with the proper namespace because for example if we're in a 'namespaced' org and navigate to the tab _without_ that namespace aligned URL, all of our imports from the schema in the web component will then not include a namespace breaking all references relying on those schema imports.

Additional details on that finding in this quip doc https://salesforce.quip.com/BNCFAalNyIqy#QFFACAw12r8

Reply:
That's a bummer, but happy to see all the collective wisdom and diligence we are gathering to inform our approach. nice!